package.json

```diff
@@ -24,6 +26,7 @@
   },
   "homepage": "https://github.com/lgeiger/node-abi#readme",
   "devDependencies": {
+    "standard": "*",
```

Comment:
I understand you here, I also set `standard` to `^a.b.c` a few month ago..
but we don't really need this